dapps/dapps_dev.json

```diff
@@ -246,11 +246,13 @@
       ]
     },
     {
-      "name": ".MEMO",
+      "name": "DotMemo",
```

Comment:
```suggestion
      "name": "Dotmemo",
```

Reply:
maybe this variant would be more recognisable? what do you think?
<img width="3024" height="1890" alt="image" src="https://github.com/user-attachments/assets/0c68ccbc-e170-4a82-ba2d-5a3addf8f479" />